sources/packages/web/src/services/AppConfigService.ts

```diff
@@ -0,0 +1,90 @@
+import AuthService from "./AuthService";
+import KeyCloak from "keycloak-js";
+
+export interface AppConfig {
+  authConfig: {
+    url: string;
+    realm: string;
+    clientId: string;
+  };
+  updateTime: Date;
+}
+
+export class AppConfigService {
+  // Share Instance
+  private static instance: AppConfigService;
+
+  private _config?: AppConfig;
+  public authService?: KeyCloak.KeycloakInstance;
+
+  private readonly _storageKey: string = "app-config";
+  private readonly _configExpiry: number = 1000 * 60 * 60;
```

Comment:
It seems that this variable is not being used. I think that the idea was to use it on line 29, right?

Reply:
Fixed